build.sh

```diff
@@ -430,7 +436,8 @@ if [ ${BUILD_BE} -eq 1 ] || [ ${BUILD_FORMAT_LIB} -eq 1 ] ; then
                   -DWITH_TENANN=${WITH_TENANN}                          \
                   -DSTARROCKS_JIT_ENABLE=${ENABLE_JIT}                  \
                   -DCMAKE_EXPORT_COMPILE_COMMANDS=ON  ..                \
-                  -DBUILD_FORMAT_LIB=${BUILD_FORMAT_LIB}
+                  -DBUILD_FORMAT_LIB=${BUILD_FORMAT_LIB}                \
+                  -DWITH_RELATIVE_SRC_PATH=${WITH_RELATIVE_SRC_PATH}
```

Comment:
didn't notice that the `..` has been moved into middle. please correct it as follows

```suggestion
                  -DCMAKE_EXPORT_COMPILE_COMMANDS=ON                  \
                  -DBUILD_FORMAT_LIB=${BUILD_FORMAT_LIB}                \
                  -DWITH_RELATIVE_SRC_PATH=${WITH_RELATIVE_SRC_PATH} \
                  ..
```

Reply:
Fixed.